src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector64.cs

```diff
@@ -2,42 +2,288 @@
+        }
+
+        public static unsafe Vector64<double> Create(double value)
+        {
+            return Unsafe.As<double, Vector64<double>>(ref value);
+        }
+
+        public static unsafe Vector64<short> Create(short value)
+        {
+            var pResult = stackalloc short[4]
+            {
+                value,
+                value,
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector64<short>>(pResult);
+        }
+
+        public static unsafe Vector64<int> Create(int value)
+        {
+            var pResult = stackalloc int[2]
+            {
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector64<int>>(pResult);
+        }
+
+        public static unsafe Vector64<long> Create(long value)
+        {
+            return Unsafe.As<long, Vector64<long>>(ref value);
+        }
+
+        [CLSCompliant(false)]
+        public static unsafe Vector64<sbyte> Create(sbyte value)
+        {
+            var pResult = stackalloc sbyte[8]
+            {
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector64<sbyte>>(pResult);
+        }
+
+        public static unsafe Vector64<float> Create(float value)
+        {
+            var pResult = stackalloc float[2]
+            {
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector64<float>>(pResult);
+        }
+
+        [CLSCompliant(false)]
+        public static unsafe Vector64<ushort> Create(ushort value)
+        {
+            var pResult = stackalloc ushort[4]
+            {
+                value,
+                value,
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector64<ushort>>(pResult);
+        }
+
+        [CLSCompliant(false)]
+        public static unsafe Vector64<uint> Create(uint value)
+        {
+            var pResult = stackalloc uint[2]
+            {
+                value,
+                value,
+            };
+
+            return Unsafe.AsRef<Vector64<uint>>(pResult);
+        }
+
+        [CLSCompliant(false)]
+        public static unsafe Vector64<ulong> Create(ulong value)
+        {
+            return Unsafe.As<ulong, Vector64<ulong>>(ref value);
+        }
+
+        public static unsafe Vector64<byte> Create(byte e0, byte e1, byte e2, byte e3, byte e4, byte e5, byte e6, byte e7)
+        {
+            var pResult = stackalloc byte[8]
+            {
+                e0,
+                e1,
+                e2,
+                e3,
+                e4,
+                e5,
+                e6,
+                e7,
+            };
+
+            return Unsafe.AsRef<Vector64<byte>>(pResult);
```

Comment:
I do not think you need to use AsRef here. I think this can be just `*(Vector64<sbyte>*)(pResult)`.

Reply:
Unfortunately, generic pointers are not supported. https://github.com/dotnet/csharplang/issues/1744 tracks adding such support.

Reply:
CC. @jaredpar